.jenkins/lsu-perftests/batch.sh

```
@@ -14,11 +14,22 @@ status_computation_and_artifacts_storage() {
     ctest_exit_code=$?
     ctest_status=$(( ctest_exit_code + configure_build_errors + test_errors + plot_errors ))
 
+    # Write status file first
+    echo "${ctest_status}" > "jenkins-hpx-${configuration_name}-ctest-status.txt"
```

Shellcheck (0.11.0)

[warning] configuration_name is referenced but not assigned.

(SC2154)